um_canonicalize/tags.yaml

```diff
@@ -185,11 +211,14 @@ categories:
   'Information Structure':
     - TOP
     - FOC
+  New:
+    - LGSPEC_AMP
   Interrogativity:
     - DECL
     - INT
   Mood:
     - IND
+    - INDF3
```

Comment:
Do you have any information on this one? I worry also this tag might be confused with `INDF` under "Definiteness".

Reply:
Looking at Swahili Wiktionary, I couldn't figure out why this tag was used.